s31052-LukaszBurnicki/zajecia3/zadanie3

```diff
@@ -0,0 +1,47 @@
+W terminalu jest kilka przydatnych skrótów klawiszowych. Jednym z nich jest
+CTRL+C lub jak to jest często podawane C-c lub ^C. Niektórzy z Państwa już
+go mieli okazję przetestować. Jest to sposób na wyłączenie aktywnego
+programu w terminalu. Proszę go przetestować w taki sposób, że:
+
+    Uruchom komendę cat bez parametrów
+
+    Wciśnij CTRL+C i zobacz co się stanie
+	przerwane zostało działanie polecenia cat
+
+Kolejnym fajnym (zależy dla kogo :) ) skrótem klawiszowym jest CTRL+D.
+Służy on do zakończenia strumienia wejściowego. Działa to trochę inaczej
+niż poprzednie rozwiązanie, mimo że na pierwszy rzut oknawygląda tak samo.
+Tym razem nie wysyła sygnału zakończenia, a jedynia zamyka strumień wejściowy.
+Jest to bardzo przydatne, jeśli chcemy zakończyć działanie jakiegoś programu
+korzystającego ze standardowego wejścia (stdin), ale w sposób możliwie bezpieczny.
+
+Zobacz co się stanie:
+
+    Wydaj komendę cat > wynik3_1.txt
+
+    Wpisz tekst witaj bez wciskania klawisza Enter
+
+    Wciśnij CTRL+C
+
+    Zobacz co się znalazło w pliku wynik3_1.txt
+	wpisany tekst po wciśnięciu CTRL+C nie został wpisany
+	zostało przerwane działanie polecenia
+
+    Wydaj komendę cat > wynik3_2.txt
+
+    Wpisz tekst witaj bez wciskania klawisza Enter
+
+    Wciśnij CTRL+D (możliwe, że będzie trzeba wcisnąć go 2x)
+
+    Zobacz co się znalazło w pliku wynik3_2.txt
+	tym razem tekst został zapisany już w pliku
+	ale pojawia się na końcu w tej samej lini z kursorem
+	CTRL+D przerywa działanie ale zapisuje rzeczy które zostały wykonane przed
```

Comment:
😡Źle
Na zajęciach explicite mówiłem, by mi takich odopowiedzi nie pisać, że ^C kończy bez zapisania a ^D z zapisaniem.
Nie. To błędna odpowiedź.
Przerwanie działania programu jest tylko efektem ubocznym akcji użytkownika ale samo ^D nie przerywa działania programu.
Do poprawy!